python/cuml/fil/fil.pyx

```diff
@@ -398,15 +406,27 @@ class ForestInference(Base):
            Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
            ndarray, cuda array interface compliant array like CuPy
            For optimal performance, pass a device array with C-style layout
-        preds: gpuarray or cudf.Series, shape = (n_samples,)
+        preds: gpuarray or cudf.Series, shape = (n_samples,) for class prediction
```

Comment:
This array doesn't necessarily store classes. It can store, e.g. probabilities (for classification), or predicted values (for regression). I think it's better to remove "for class prediction", and leave the description of this parameter as is.